packages/tokens/src/breakpoints.ts

```diff
@@ -0,0 +1,2 @@
+const breakpoints = [320, 768, 1024, 1440];
```

Comment:
Can we change this to an enum so we can describe each breakpoint? Something like `phone`, `tablet`, `desktop`, `largeDesktop`

Reply:
yes that probably makes more sense... 🤦